Makefile

```diff
@@ -15,9 +18,21 @@ build:
 	@docker build -t $(IMAGE_NAME):latest -f dockerfiles/Dockerfile .
 	@docker tag $(IMAGE_NAME):latest $(IMAGE_NAME):$(IMAGE_TAG)
 
+build-dbs:
```

Comment:
Nit: maybe call this `build-utility-images` for clarity? Similarly for `push-utility-images` vs `push-dbs`

Reply:
Roger that